docs/en_us/developers/source/accessibility.rst

```diff
@@ -0,0 +1,164 @@
+#######################
+edX Accessibility Guide
+#######################
+
+EdX measures and evaluates accessibility using the World Wide Web Consortium's `Web Content Accessibility Guidelines (WCAG) 2.0 <http://www.w3.org/TR/WCAG/>`_ (Dec. 11, 2008).  All features merged into edX are expected to `conform <http://www.w3.org/TR/WCAG20/#conformance>`_ to `Level AA <http://www.w3.org/TR/WCAG20/#cc1>`_ of this specification.  This guide is intended to extend the guidance available in WCAG 2.0, with a focus on features frequently found in Open edX platform as well as Learning and Content Management Systems in general.
+
+Introduction
+************
+
+The core mission of edX is to expand access to education for everyone.  We expect that any user interfaces that are developed for the Open edX platform can be used by everyone, regardless of any physical limitation they may have.  The Open edX platform is used everyday by people who may not be able to see, hear or have the dexterity to use traditional modes of computer interaction (like the mouse).  Understanding a few core concepts about how people with disabilities use the web and web applications should make the guidance in this document a little easier to understand.
```

Comment:
Starting a new paragraph at "The Open edX platform is used every day" might make your lead-in sentence to the bullet list more obvious? I feel as if there could be a clearer link between the last sentence and the bullet points, but can't come up with anything yet... will give it some though.